packages/esm-patient-chart-app/src/config-schema.ts

```diff
@@ -42,6 +42,11 @@ export const esmPatientChartSchema = {
     _description: 'Custom label to use on the start visit actions',
     _default: '',
   },
+  defaultPatientIdentifier: {
```

Comment:
General question: What is the reason to duplicate all this and have it in patient-chart and patient-banner? Should we maybe have only patient-banner? There might be some reason behind it, but I fail to see / remember it.

cc @denniskigen

Reply:
Me neither @FlorianRappl, I was very surprised when I first saw this duplication. I intend to clean it up soon.